src/LowHash0.cpp

```diff
@@ -573,7 +617,9 @@ void LowHash0::computeBucketHistogram()
     for(const vector<uint64_t>& histogram: threadBucketHistogram) {
         largestBucketSize = max(largestBucketSize, uint64_t(histogram.size()));
     }
-    vector<uint64_t> bucketHistogram(largestBucketSize, 0);
+    
+    bucketHistogram.resize(largestBucketSize, 0);
```

Comment:
This only sets to zero any values that were not previously present. To make sure the entire vector is zeroed, you need to also use `std::fill`.